common/source/docs/common-running-without-rc.rst

```diff
@@ -0,0 +1,40 @@
+.. _common-running-without-rc:
+
+========================================
+Running ArduPilot without an RC Receiver
+========================================
+
+This article provides an overview of how ArduPilot can be configured
+to safely run with no RC receiver attached.
+
+.. note::
+
+   Fitting your vehicle with an RC receiver allows for additional recovery measures should your vehicle misbehave.  Particularly while creating and developing a vehicle we strongly recommend flying with an RC receiver/transmitter pair providing this additional recovery mechanism.
+
+  Before making the decision to configure your vehicle without an RC receiver, consider options such as ignoring RC failure for failsafe purposes; see "Configuring for Emergency RC Receiver Use", below.
```

Comment:
I don't see a section marked "Configuring for Emergency RC Receiver Use" below

Reply:
> I don't see a section marked "Configuring for Emergency RC Receiver Use" below

Yeah, this is a WIP... patches against it welcome.  It requires a chunk of research and *must* be right....